### DISCUSSION TITLE
64 bit Image

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#460] [General]
@rern 

I have asked before, but I would really like to know if we are still supposed to be using the 64 bit image? 

Is a 64 bit download still available?

Using the same image for different versions of the Pi would seem to be a big advantage, so what is the advantage of having separate 32 bit images?

Either image works well for me, so thanks for creating a brilliant system. I would just like to know which image has a future.

**Comment by rern** [comment]
It's the 'audio glitches on refresh' issue with the 64bit version. Anyone who is currently running it please share whether it's still there.

Of course single image for multiple Pis will be much better. But there'll still be 3 versions, for 0-1, 2(32bit) and 2(64bit)-3-4, anyway.

**Reply by Roland-B** [reply]
@rern No glitches for me on 64-Bit version.

@R101-Github Available on releases page. Look out for releases with "(6) Assets".

Here is link to the latest if helps:
https://github.com/rern/rAudio-1/releases/download/i20210716/rAudio-1-RPi64-20210716.img.xz

RolandB

rAudio 1 • 20210807
5.10.52-1-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Reply by R101G** [reply]
@rern 
No glitches for me either, running 64 bit with local browser disabled on a Pi4. I can try it on a Pi3 if required.

@Roland-B 
Thanks - it never seems easy to find!

**Reply by EarsofTin** [reply]
Never had a problem

rAudio 1 • 20210807
5.10.27-3-ARCH aarch64
Raspberry Pi 3 Model B Plus Rev 1.3
BCM2837B0 • 953MiB
4 x Cortex-A53 @ 1.4GHz

**Reply by klassikmann** [reply]
rAudio 1 • 20210807
5.10.52-6-ARCH aarch64
Raspberry Pi 3 Model B Rev 1.2

Using 64-bit images since first availability: No problems at all.

**Reply by jay77777777** [reply]
Another vote for zero issues with the 64 bit version. Running it on a RP3B+ and RP4
